pointmatcher/InspectorsImpl.cpp

```diff
@@ -320,25 +320,42 @@ void InspectorsImpl<T>::AbstractVTKInspector::dumpDataLinks(
 	}
 	const int knn = matches.ids.rows();
 	
-	stream << "LINES " << readingPtCount*knn << " "  << readingPtCount*knn * 3 << "\n";
+	size_t matchCount = readingPtCount*knn;
+	for(int k = 0; k < knn; ++k)
+	{
+		for (int i = 0; i < readingPtCount; ++i)
+		{
+			if(matches.ids(k, i) == Matches::InvalidId){
```

Comment:
This is a detail but the style (space after `if` and `{` at the end of the line) does not match the rest of the code.